src/collections/communities/poodle-hub.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Poodle Hub
+description: A group devoted to the making and playing of Poodles with a completive
```

Comment:
I'm not sure what a "completive ranked pool" is. Does that make sense and I just don't get it or is that a typo perhaps and it should have been competitive?

Reply:
Checked with submitter and changed to "competitive"